src/index.tsx

```diff
@@ -479,6 +481,40 @@ const KEYS = {
   BACKSPACE: "Backspace"
 };
 
+const SHAPES = [
+  {
+    label: "Rectange",
+    value: "rectangle"
+  },
+  {
+    label: "Ellipse",
```

Comment:
This could be either Ellipse (Figma) or Oval (Sketch). I left it unchanged for now

Reply:
I have no preference either way. Roughjs calls them ellipse so I used that name.